raiden/raiden_service.py

```diff
@@ -316,26 +316,21 @@ def start(self):
                     log.error(str(e))
 
         self.alarm.start()
+        self.transport.start(self)
 
         events_queues = views.get_all_messagequeues(chain_state)
-        queueids_to_queues = dict()
 
         for queue_identifier, event_queue in events_queues.items():
+            self.transport.start_health_check(queue_identifier.recipient)
```

Comment:
this is made a noop in a follow up PR:

https://github.com/raiden-network/raiden/pull/2309/files#diff-72b8041548bf416a299ac073c02e24e5R447

if you want to do this here, please provide a different solution for the stress test.

Reply:
@hackaugusto why is this a noop? I moved `start` to before these messages, the transport should already be started at this point. `send_async` should only be called with health-checked nodes, that's why I `start_health_check` before `send_async`ing the messages in the queue